Makefile

```diff
@@ -103,3 +103,22 @@ start-service:
 .PHONY: stop-service
 stop-service:
 	docker compose down
+
+# Check that docker is installed.
+.PHONY: check-docker-tool-check
+check-docker-tool-check:
+	@docker --version > /dev/null
```

Comment:
can we make it print a note that tool X needs to be installed if it fails?

Reply:
Took care of it. Thanks